.circleci/config.yml

```diff
@@ -364,11 +367,73 @@ commands:
           det-version: <<parameters.det-version>>
           agent-instance-type: <<parameters.agent-instance-type>>
           max-dynamic-agents: <<parameters.max-dynamic-agents>>
-      - set-master-address:
+      - set-master-address-aws:
           cluster-id: ${CLUSTER_ID}
       - wait-for-master:
           host: ${MASTER_HOST}
 
+  terminate-gke-cluster:
+    parameters:
+      cluster-id:
+        type: string
+    steps:
+      - run:
+          name: Terminate GKE Cluster
+          when: always
+          command: |
+            gcloud container clusters delete <<parameters.cluster-id>> --quiet --region=us-west1
```

Comment:
question: why not using `gke/delete-cluster`?

Reply:
Good question. When using orbs, it's not possible to specify the `when` command, and we need to run even the tests fail.

Reply:
I see. Seems a gotcha. Can you put that in a comment? Thanks

Reply:
Yeah definitely